cmd/gaia/cmd/gaiad/main.go

```diff
@@ -55,7 +55,7 @@ func main() {
 
 	// prepare and add flags
 	executor := cli.PrepareBaseCmd(rootCmd, "GA", app.DefaultNodeHome)
-	rootCmd.Flags().BoolVar(&assertInvariantsBlockly, flagAssertInvariantsBlockly,
+	rootCmd.PersistentFlags().BoolVar(&assertInvariantsBlockly, flagAssertInvariantsBlockly,
```

Comment:
whoops - good thing this was caught ahhhhh. (you wouldn't been able to enable invariants with the start command otherwise)